Sources/ComposableArchitecture/TestStore.swift

```diff
@@ -529,43 +529,6 @@ public final class TestStore<State, Action> {
     self.useMainSerialExecutor = true
   }
 
-  /// Creates a test store with an initial state and a reducer powering its runtime.
-  ///
-  /// See <doc:Testing> and the documentation of ``TestStore`` for more information on how to best
-  /// use a test store.
-  ///
-  /// - Parameters:
-  ///   - initialState: The state the feature starts in.
-  ///   - reducer: The reducer that powers the runtime of the feature.
-  ///   - prepareDependencies: A closure that can be used to override dependencies that will be
-  ///     accessed during the test. These dependencies will be used when producing the initial
-  ///     state.
-  @available(*, deprecated, message: "State must be equatable to perform assertions.")
```

Comment:
Unrelatedly, this deprecated overload of `TestStore.init` has started to cause diagnostics problems in Xcode. It was introduced before 1.0.0 when test stores could be scoped, but is no longer necessary or useful, so let's drop it entirely.